pkgs/racket-doc/scribblings/reference/eval-model.scrbl

```diff
@@ -118,18 +119,19 @@ branch of an @racket[if] form is also in @tech{tail position}.
 
 @tech{Tail-position} specifications provide a guarantee about the
 asymptotic space consumption of a computation. In general, the
-specification of @tech{tail positions} goes with each syntactic form,
-like @racket[if].
+specification of @tech{tail positions} accompanies the description of
+each syntactic form, such as @racket[if].
 
 @;------------------------------------------------------------------------
 @section[#:tag "values-model"]{Multiple Return Values}
 
-A Racket expression can evaluate to @deftech{multiple values}, in the
-same way that a procedure can accept multiple arguments.
+A Racket expression can evaluate to @deftech{multiple values}, similar
+to the way that a procedure can accept multiple arguments.
```

Comment:
I think this is a good change, but I think it could be made even clearer. Maybe something like “to provide symmetry with the fact that a procedure can accept multiple arguments”?